ansible/install_python.yml

```diff
@@ -1,8 +1,7 @@
 ---
 # based on https://gist.github.com/gwillem/4ba393dceb55e5ae276a87300f6b8e6f
 - hosts: all
-  become: true
   gather_facts: false
   tasks:
   - name: Install Python 2
-    raw: test -e /usr/bin/python || (apt-get -y update && apt-get install -y python-minimal)
+    raw: test -e /usr/bin/python || (sudo apt-get -y update && sudo apt-get install -y python-minimal)
```

Comment:
Possible this can be changed back... Ansible behaves differently (at least through Packer) when it was being run in the container as `root`.